src/main/groovy/org/web3j/solidity/gradle/plugin/SolidityCompiler.groovy

```diff
@@ -60,6 +64,10 @@ class SolidityCompiler {
         }
     }
 
+    static boolean checkWindowsVcppExists(String pathVariable) {
+        return Arrays.stream(pathVariable.split(";")).anyMatch({ p -> new File(p + File.separator + "vcruntime140.dll").exists() })
+    }
```

Comment:
Is there any reason apart from unit testing that this is package is protected?

Reply:
is package protected for unit test only